packages/blockchain-wallet-v4-frontend/src/scenes/Login/template.js

```diff
@@ -128,12 +129,30 @@ const Login = (props) => {
         { authType > 0 &&
           <FormGroup>
             <FormItem>
-              <FormLabel for='code'>
-                { authType === 1 && <FormattedMessage id='scenes.login.yubikey' defaultMessage='Yubikey' /> }
-                { authType === 4 && <FormattedMessage id='scenes.login.google' defaultMessage='Authenticator App Code' /> }
-                { authType === 5 && <FormattedMessage id='scenes.login.mobile' defaultMessage='SMS Code' /> }
-              </FormLabel>
-              <Field name='code' validate={[required]} component={TextBox} borderColor={twoFactorError ? 'invalid' : undefined} />
```

Comment:
Out of curiosity does `authType === 1 ? PasswordBox : TextBox` work?

Reply:
If not you could create a function probably that does return the correct component instead of 3 cases

Reply:
fixed